.prettierrc

```diff
@@ -1,9 +1,7 @@
 {
-  "bracketSpacing": false,
```

Comment:
The files in this project seemed to have inconsistent linting applied. I've updated the rules to match bulletweb, but I'm fine with whatever consistent rules you want here.